<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTimeWidget</class>
 <widget class="QWidget" name="MyTimeWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MyTimeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>160</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
      </font>
     </property>
     <property name="title">
      <string>Date/Time Edit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnReadTime">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ReadSystemTime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSystemTime">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnSetTime">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SetWidgetTime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei UI Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="displayFormat">
           <string>yyyy/M/d H:mm:ss</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Timer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Timer Period: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxTimerPeriod">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Time Elapsed: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTimeElapsed">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::TopToBottom</enum>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTimer1Count">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>24</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnStartTimer">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStopTimer">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI Light</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
